rskj-core/src/main/java/co/rsk/peg/BridgeStorageProvider.java

```diff
@@ -462,19 +463,41 @@ public void setPendingFederation(PendingFederation federation) {
      */
     public void savePendingFederation() {
         if (shouldSavePendingFederation) {
-            RepositorySerializer<PendingFederation> serializer = BridgeSerializationUtils::serializePendingFederationOnlyBtcKeys;
-
-            if (activations.isActive(RSKIP123)) {
-                // we only need to save the standard part of the fed since the emergency part is constant
-                saveStorageVersion(
-                    PENDING_FEDERATION_FORMAT_VERSION.getKey(),
-                    STANDARD_MULTISIG_FEDERATION.getFormatVersion()
-                );
-                serializer = BridgeSerializationUtils::serializePendingFederation;
+            if (!activations.isActive(RSKIP123)) {
```

Comment:
```suggestion
            if (activations.isActive(RSKIP123)) {
```
It's clearer to ask for the positive case first